### PR TITLE
Update version to 2.0.0

### PR DESCRIPTION
**Pull request by JafarAbdi**
### Description

Please explain the changes you made, including a reference to the related issue if applicable

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Extend the tutorials / documentation [reference](http://moveit.ros.org/documentation/contributing/)
- [ ] Document API changes relevant to the user in the MIGRATION.md notes
- [ ] Create tests, which fail without this PR [reference](https://ros-planning.github.io/moveit_tutorials/)
- [ ] Include a screenshot if changing a GUI
- [ ] While waiting for someone to review your request, please help review [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: moveit_core/CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,36 @@
...
+* Upgrade to ROS 2 Eloquent (`#152 <https://github.com/JafarAbdi/moveit2/issues/152>`_)
+* Fix unit tests
+* Remove redundant build dependency to 'angles'
+* Fix CMakeLists.txt files for Eloquent
+* Pass node to KinematicsBase initialization (`#145 <https://github.com/JafarAbdi/moveit2/issues/145>`_)
+* replace rosunit -> ament_cmake_gtest
+* Port planning_request_adapter_plugins to ROS2 (`#114 <https://github.com/JafarAbdi/moveit2/issues/114>`_)
+* Port planning_request_adapter_plugins to ROS 2
+* Fix ament_lint
+* Contributors: Henning Kayser, Jafar Abdi, JafarAbdi
```
Comment (henningkayser) [inline moveit_core/CHANGELOG.rst]:
```suggestion
* Contributors: Henning Kayser, Jafar Abdi
```

File: moveit_ros/visualization/CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,26 @@
 Changelog for package moveit_ros_visualization
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+2.0.0 (2020-02-17)
+------------------
+* moveIt_ros_visualization fixes (`#168 <https://github.com/JafarAbdi/moveit2/issues/168>`_)
```
Comment (henningkayser) [inline moveit_ros/visualization/CHANGELOG.rst]:
```suggestion
* moveit_ros_visualization fixes (`#168 <https://github.com/JafarAbdi/moveit2/issues/168>`_)
```

File: moveit_ros/planning/CHANGELOG.rst [hunk 3]
```diff
@@ -2,6 +2,71 @@
...
+* Enable planning_request_adapter_plugins for colcon build
+* Migrate to ROS 2 Logger
+* Update CMakeLists.txt
+* Port planning_request_adapter_plugins to ROS 2
+* Port rdf_loader to ROS2 (`#104 <https://github.com/JafarAbdi/moveit2/issues/104>`_)
+* rdf_loader: Port to ROS2
+* rdf_loader: Migrate CMakeLists.txt to ament_cmake
+* moveit_ros_planning: Enable rdf_loader
+* moveit_ros_planning: Migrate package.xml
+* Contributors: Henning Kayser, Jafar Abdi, JafarAbdi, Robert Haschke, RoboticsYY, Víctor Mayoral Vilches, Yu, Yan
```
Comment (henningkayser) [inline moveit_ros/planning/CHANGELOG.rst]:
```suggestion
* Contributors: Henning Kayser, Jafar Abdi, Robert Haschke, Yu Yan, Víctor Mayoral Vilches
```

File: moveit_ros/occupancy_map_monitor/CHANGELOG.rst [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+* Port occupancy_map_monitor to ROS2 (`#148 <https://github.com/JafarAbdi/moveit2/issues/148>`_)
+* Using THROTTLE macros
+* Port occupancy_map_server
+* Port occupancy_map_monitor
+* Port occupancy_map_updater
+* Enable build of occupancy_map_monitor
+* replace rosunit -> ament_cmake_gtest
+* Port rdf_loader to ROS2 (`#104 <https://github.com/JafarAbdi/moveit2/issues/104>`_)
+* Enable moveit_ros_planning for colcon build
+* Contributors: 7929f1cfe06ad78d8b8c60a417e1b4a8de37050c, Henning Kayser, JafarAbdi, RoboticsYY
```
Comment (henningkayser) [inline moveit_ros/occupancy_map_monitor/CHANGELOG.rst]:
```suggestion
* Contributors: 7929f1cfe06ad78d8b8c60a417e1b4a8de37050c, Henning Kayser, Jafar Abdi, Yu Yan
```

File: moveit_plugins/moveit_simple_controller_manager/CHANGELOG.rst [hunk 5]
```diff
@@ -2,6 +2,15 @@
...
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+2.0.0 (2020-02-17)
+------------------
+* MoveItSimpleControllerManager refactor parameter lookup
+* Fix plugin install of MoveItSimpleControllerManager
+* Port moveit_simple_controller_manager to ROS 2 (`#158 <https://github.com/JafarAbdi/moveit2/issues/158>`_)
+* Test logger in header file
+* Port moveit_simple_controller_manager
+* Contributors: Henning Kayser, JafarAbdi
```
Comment (henningkayser) [inline moveit_plugins/moveit_simple_controller_manager/CHANGELOG.rst]:
```suggestion
* Contributors: Henning Kayser, Jafar Abdi
```

File: moveit_kinematics/CHANGELOG.rst [hunk 6]
```diff
@@ -2,6 +2,16 @@
...
 
+2.0.0 (2020-02-17)
+------------------
+* Port moveit kinematics to ROS2 (`#128 <https://github.com/JafarAbdi/moveit2/issues/128>`_)
+* moveit_kinematics: Port unit tests
+* Port srv_kinematics_plugin to ROS2
+* Port lma_kinematics_plugin to ROS2
+* Port kdl_kinematics_plugin to ROS2
+* Enable moveit_kinematics & Port CMakeLists.txt + package.xml
+* Contributors: Henning Kayser, JafarAbdi
```
Comment (henningkayser) [inline moveit_kinematics/CHANGELOG.rst]:
```suggestion
* Contributors: Henning Kayser, Jafar Abdi
```

File: moveit_demo_nodes/run_moveit_cpp/CHANGELOG.rst [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+* ROS2 demo: Improve plan visualization
+* Add .repos with demo runtime dependencies
+* Add trajectory publisher
+* Add fake joint driver node with config files
+* Edit Rviz config file
+* ROS2 demo: Add RViz config file
+* ROS2 demo: Async execution and plan visualization
+* ROS2 demo: Launch RViz
+* Add demo package run_moveit_cpp
+* Contributors: Henning Kayser, JafarAbdi
```
Comment (henningkayser) [inline moveit_demo_nodes/run_moveit_cpp/CHANGELOG.rst]:
```suggestion
* Contributors: Henning Kayser, Jafar Abdi
```

File: moveit_ros/visualization/CHANGELOG.rst [hunk 8]
```diff
@@ -2,6 +2,26 @@
...
+* Port moveit ros visualization to ROS 2 (`#160 <https://github.com/JafarAbdi/moveit2/issues/160>`_)
+  Port moveit ros visualization
+* Port planning_scene_rviz_plugin to ROS2
+* Port robot_state_rviz_plugin to ROS2
+* Port rviz_plugin_render_tools to ROS2
+* Enable moveit_ros_visualization & Port CMakeLists.txt + package.xml
+* Port external Rviz mesh_shape class and add them to MoveIt 2 project temporary
+* Port rdf_loader to ROS2 (`#104 <https://github.com/JafarAbdi/moveit2/issues/104>`_)
+* Enable moveit_ros_planning for colcon build
+* Contributors: Henning Kayser, JafarAbdi
```
Comment (henningkayser) [inline moveit_ros/visualization/CHANGELOG.rst]:
```suggestion
* Contributors: Henning Kayser, Jafar Abdi
```

File: moveit_ros/planning_interface/CHANGELOG.rst [hunk 9]
```diff
@@ -2,6 +2,20 @@
...
+* Fix parameter lookup and runtime in MoveItCpp
+* Fix moveit_ros_visualization (`#167 <https://github.com/JafarAbdi/moveit2/issues/167>`_)
+* moveit_ros_planning_interface: Fix libraries export
+* Port moveit_cpp to ROS 2 (`#163 <https://github.com/JafarAbdi/moveit2/issues/163>`_)
+* Port common_planning_interface_objects to ROS 2 (`#159 <https://github.com/JafarAbdi/moveit2/issues/159>`_)
+* Port common_planning_interface_objects to ROS2
+* Enable moveit_ros_planning_interface & Port CMakeLists.txt + package.xml
+* Port rdf_loader to ROS2 (`#104 <https://github.com/JafarAbdi/moveit2/issues/104>`_)
+* Enable moveit_ros_planning for colcon build
+* Contributors: Henning Kayser, JafarAbdi
```
Comment (henningkayser) [inline moveit_ros/planning_interface/CHANGELOG.rst]:
```suggestion
* Contributors: Henning Kayser, Jafar Abdi
```

File: moveit_core/CHANGELOG.rst [hunk 10]
```diff
@@ -2,6 +2,36 @@
...
+* Load planner parameters from subnamespace
+* Port moveit_planners_ompl to ROS 2 (`#142 <https://github.com/JafarAbdi/moveit2/issues/142>`_)
+* Load OMPL planner config parameters
+* Re-enable parameter util functions
+  * Construct constraints from node parameters
+* Compile OMPL planning plugin
+* Fix Eigen3 linking issue in moveit_core (`#164 <https://github.com/JafarAbdi/moveit2/issues/164>`_)
+  * Closes `#151 <https://github.com/JafarAbdi/moveit2/issues/151>`_
+* Port moveit ros visualization to ROS 2 (`#160 <https://github.com/JafarAbdi/moveit2/issues/160>`_)
+  Port moveit ros visualization
```
Comment (henningkayser) [inline moveit_core/CHANGELOG.rst]: There is some redundancy in here

File: moveit_core/CHANGELOG.rst [hunk 11]
```diff
@@ -2,6 +2,34 @@
 Changelog for package moveit_core
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+2.0.0 (2020-02-17)
+------------------
+* [fix] Fix double node executor exceptions
```
Comment (davetcoleman) [inline moveit_core/CHANGELOG.rst]: i don't think the brackets are absolutely necessary, but they are nice.

File: moveit_demo_nodes/run_moveit_cpp/package.xml [hunk 12]
```diff
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>run_moveit_cpp</name>
```
Comment (davetcoleman) [inline moveit_demo_nodes/run_moveit_cpp/package.xml]: this package naming is a break from our previous convention of always prepending with ``moveit_*``... it feels messy. Should be ``moveit_cpp_demo`` IMHO

Also needs a pkg description.

Comment (henningkayser) [reply moveit_demo_nodes/run_moveit_cpp/package.xml:4]: Will address in the follow-up

**Pull request merged**